src/components/VSnackbar.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div :class="[$style.snackbar, $style[size]]">
+    <!-- @slot Your message or subcomponent goes passed here. -->
+    <slot />
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType } from '@nuxtjs/composition-api'
+
+export default defineComponent({
+  name: 'VSnackbar',
+  props: {
+    size: {
+      type: String as PropType<'small' | 'large'>,
+      default: 'small',
+    },
+  },
+})
+</script>
+
+<style module>
```

Comment:
This might be a personal preference but I generally prefer to apply styles using `computed` values in the component rather than `<style>` tags.